app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: str) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result_list = [Person(name=each_one["name"], age=each_one["age"])
+                   for each_one in people]
```

Comment:
Good one, just would be great to fix indentation a bit
```suggestion
    result_list = [
        Person(name=each_one["name"], age=each_one["age"]) 
        for each_one in people
    ]
```

Reply:
Also does it make sense to use `person` Instead of `each_one`?

Reply:
Yep, you are right. 
Renamed and fixed iteration issue.